src/gentropy/datasource/gwas_catalog/study_index.py

```diff
@@ -332,7 +330,6 @@ def from_source(
         return (
             cls._parse_study_table(catalog_studies)
             .annotate_ancestries(ancestry_file)
-            .annotate_sumstats_info(sumstats_lut)
```

Comment:
This change can be problematic. Not on top-hits, or sumstats processing but for the generation of the curation table. @DSuveges we can discussed but I think @addramir was suggesting to not care too much about curation for now